initial_setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# this script performs the initial setup
+
+URL="http://0.0.0.0:8000/"
+status_code=000
+ENV_EXAMPLE=".env.example"
+ENV_ORIG=".env"
+TIMEOUT_TIME=60
+
+error_check() {
+    if [ $? -ne 0 ]; then
+        echo "FAILED"
+        exit
+    fi
+}
+
+# instantiate .env file
+cp ${ENV_EXAMPLE} ${ENV_ORIG}
+
+# runs docker compose as daemon process
+docker-compose up -d
+error_check
+
+# loop until http://0.0.0.0:8000/ is not active and
+# timeout does not occurs
+while [[ ${status_code} -ne 302 && ${SECONDS} -lt ${TIMEOUT_TIME} ]];
+do
+    status_code=$(curl --write-out %{http_code} --silent --output /dev/null ${URL})
```

Comment:
What's `http_code` here? Is that a typo for `status_code`?

Reply:
Refer to [curl's man page](https://man7.org/linux/man-pages/man1/curl.1.html). See the `--write-out` option.
>**http_code**
> The numerical response code that was found in the last retrieved HTTP(S) or FTP(s) transfer.